src/Selenium2Library/keywords/_element.py

```diff
@@ -734,6 +791,79 @@ def _map_ascii_key_code_to_key(self, key_code):
             key = chr(key_code)
         return key
 
+    def _map_named_key_code_to_special_key(self, key_name):
+        map = {
+            'ADD': Keys.ADD,
+            'ALT': Keys.ALT,
+            'ARROW_DOWN': Keys.ARROW_DOWN,
+            'ARROW_LEFT': Keys.ARROW_LEFT,
+            'ARROW_RIGHT': Keys.ARROW_RIGHT,
+            'ARROW_UP': Keys.ARROW_UP,
+            'BACKSPACE': Keys.BACKSPACE,
+            'BACK_SPACE': Keys.BACK_SPACE,
+            'CANCEL': Keys.CANCEL,
+            'CLEAR': Keys.CLEAR,
+            'COMMAND': Keys.COMMAND,
+            'CONTROL': Keys.CONTROL,
+            'DECIMAL': Keys.DECIMAL,
+            'DELETE': Keys.DELETE,
+            'DIVIDE': Keys.DIVIDE,
+            'DOWN': Keys.DOWN,
+            'END': Keys.END,
+            'ENTER': Keys.ENTER,
+            'EQUALS': Keys.EQUALS,
+            'ESCAPE': Keys.ESCAPE,
+            'F1': Keys.F1,
+            'F2': Keys.F2,
+            'F3': Keys.F3,
+            'F4': Keys.F4,
+            'F5': Keys.F5,
+            'F6': Keys.F6,
+            'F7': Keys.F7,
+            'F8': Keys.F8,
+            'F9': Keys.F9,
+            'F10': Keys.F10,
+            'F11': Keys.F11,
+            'F12': Keys.F12,
+            'HELP': Keys.HELP,
+            'HOME': Keys.HOME,
+            'INSERT': Keys.INSERT,
+            'LEFT': Keys.LEFT,
+            'LEFT_ALT': Keys.LEFT_ALT,
+            'LEFT_CONTROL': Keys.LEFT_CONTROL,
+            'LEFT_SHIFT': Keys.LEFT_SHIFT,
+            'META': Keys.META,
+            'MULTIPLY': Keys.MULTIPLY,
+            'NULL': Keys.NULL,
+            'NUMPAD0': Keys.NUMPAD0,
+            'NUMPAD1': Keys.NUMPAD1,
+            'NUMPAD2': Keys.NUMPAD2,
+            'NUMPAD3': Keys.NUMPAD3,
+            'NUMPAD4': Keys.NUMPAD4,
+            'NUMPAD5': Keys.NUMPAD5,
+            'NUMPAD6': Keys.NUMPAD6,
+            'NUMPAD7': Keys.NUMPAD7,
+            'NUMPAD8': Keys.NUMPAD8,
+            'NUMPAD9': Keys.NUMPAD9,
+            'PAGE_DOWN': Keys.PAGE_DOWN,
+            'PAGE_UP': Keys.PAGE_UP,
+            'PAUSE': Keys.PAUSE,
+            'RETURN': Keys.RETURN,
+            'RIGHT': Keys.RIGHT,
+            'SEMICOLON': Keys.SEMICOLON,
+            'SEPARATOR': Keys.SEPARATOR,
+            'SHIFT': Keys.SHIFT,
+            'SPACE': Keys.SPACE,
+            'SUBTRACT': Keys.SUBTRACT,
+            'TAB': Keys.TAB,
+            'UP': Keys.UP
+        }
+        key = map.get(key_name)
```

Comment:
I know I mentioned this in the last PR, but you should probably replace this whole map with 

```python
try:
  return getattr(Keys, key_name)
except:
  # emit proper error here
```

That way you don't have to define this whole redundant map. It's smaller and less maintenance as well. 

Oh, and you should probably give a descriptive error on the assert just so people know why it failed.